packages/compare/src/utils/markdown.ts

```diff
@@ -0,0 +1,19 @@
+export function header1(text: string) {
+  return `# ${text}\n`;
+}
+
+export function header2(text: string) {
+  return `## ${text}\n`;
+}
+
+export function header3(text: string) {
+  return `### ${text}\n`;
+}
```

Comment:
These seem to rely on the result being prefixed with a newline. Fortunately that appears to be the case everywhere it's currently used, but that probably at least deserves a comment if not a more robust implementation.